pensjon-brevbaker/src/main/kotlin/no/nav/pensjon/brev/maler/fraser/vedlegg/VedleggOpplysningerOmEtteroppgjoeret.kt

```diff
@@ -706,6 +654,24 @@ data class OmBeregningAvUfoeretrygd(
                 ) + " calculation",
             )
         }
+
+        paragraph {
+            val inntektFoerFratrekk = pensjonsgivendeInntekt.inntekt.sum.format()
+            textExpr(
+                Bokmal to "I ".expr() + periode.format() + " var den pensjonsgivende inntekten din " + inntektFoerFratrekk + " kroner.",
+                Nynorsk to "I ".expr() + periode.format() + " var den pensjonsgivande inntekta di " + inntektFoerFratrekk + " kroner.",
+                English to "In ".expr() + periode.format() + ", your pensionable income was NOK " + inntektFoerFratrekk + ".",
+            )
+            showIf(pensjonsgivendeInntekt.fratrekk.fratrekk.isNotEmpty()) {
```

Comment:
Dette er ikke noe vi trenger å gjøre noe med nå, men det er noe jeg mener vi bør jobbe for å få til og det er noe som må inn på pesys-siden. Her kommer vi med en påstand om data consistency, nemlig at om pensjonsgivende-fratrekk-listen er tom så er pensjonsgivendeInntektBruktIBeregningen lik pensjonsgivende-inntekt-sum. Det burde vi da asserte på tolkning siden, slik at malen ikke gjør antagelser om data som vi ikke vet stemmer. Hva tenker du  @hconhegg?